Source/JNA/waffle-jna/pom.xml

```diff
@@ -80,5 +80,11 @@
             <artifactId>caffeine</artifactId>
             <version>${caffeine.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>RELEASE</version>
```

Comment:
@pedroneil Can you add proper version of mockito in property section?  The RELEASE and others like it are deprecated and considered bad practice.  Thanks.

Reply:
busy doing that